katas/content/teleportation/entangled_pair/index.md

```diff
@@ -0,0 +1,5 @@
+**Input:** 
+Two qubits qAlice and qBob, each in $|0\rangle$ state.
+
+**Goal:**  
+Prepare a Bell state $|\Phi^{+}\rangle = \frac{1}{\sqrt{2}}(|00\rangle + |11\rangle)$ on these qubits.
```

Comment:
Let's start using the \ket macro in this kata instead of | ... \rangle. We will get to https://github.com/microsoft/qsharp/issues/1448 at some point, but there's no need to increase the amount of work that needs to be done there :-)